scripts/release/release-test.R

```diff
@@ -491,6 +491,12 @@ if(url.exists(armadillo_url)) {
   exit_test(msg)
 }
 
+as_docker_container <- FALSE
+if("Y" == Sys.getenv("AS_DOCKER_CONTAINER", "N")) {
+  as_docker_container <- TRUE
+}
+cli_alert_info(sprintf("Running in docker container %d", as.integer(as_docker_container)))
+
```

Comment:
This is a weird name variable ... it indicates not to use the API calls for creating R containers.

This needs a new name